webui/apps/docs/src/content/post/sql_agent.mdx

```diff
@@ -0,0 +1,62 @@
+---
+publishDate: 2024-10-01T00:00:00Z
+author: 'Luke Lalor (luke@augustdata.ai)'
+category: 'Agents'
+title: "SQL Generation: An Agentic Approach"
+excerpt: "OpenAI's new model, O1, is shaking up the AI world. Learn how Eidolon is smoothing out the bumps and keeping your projects on track."
+image: ~/assets/images/agentic_sql.webp
+tags:
+  - SQL Agent
+  - LLM App Development
+  - SQL
+  - Agentic SQL
+metadata:
+  canonical: https://www.eidolonai.com/sql_agent
```

Comment:
We should reserve https://www.eidolonai.com/sql_agent for a page with more durability than a blog post. 

How about /blog/sql_query

Reply:
our blogs are hosted as top level urls

we could change that and make them all /blog/... , but is work we should do separately. I don't want to break from convention when writing an individual article.

With that said, we can rename this page. /agentic_sql would also fit if you would prefer